introduction/quickstart.md

```diff
@@ -39,10 +37,10 @@ The response will look like this :
 }
 ```
 
-the `uid` is the `:index` identifier used in all `indexes/:index` routes.
+The `uid` is the `:index` identifier used in all `indexes/:index` routes.
```

Comment:
Don't you think it would be better to change `:index` into `:index-uid`/`:uid`?

Reply:
Good point, I add it to the to-do.